Bump max version of ServiceFabric.Actors to 5.0.514

JelmerBaas opened the pull request:
Tiny change to include 5.0.514.

loekd commented:
Hi @JelmerBaas, thanks for your contribution!
Before this PR can be merged, a couple of checks and changes need to be made:
- moving from SF v4.x to v5.x indicates breaking changes. 
  - one method causes a compile issue at v5; `void ReportCompletion()` is added to `IStatelessServicePartition` so it needs to be added to `MockStatelessServicePartition`
- the package version needs to be bumped, so a new NuGet package appears. I believe this package's major version should follow the SF major version. So let's go for 5.0.0. (see readme for details)
- instead of limiting the package version to 5.0.514, let's go for 6.0.0 instead, so we can allow the entire range of upcoming non-breaking changes all the way up to v6.

Are you comfortable making these changes, or would you like me to help?

JelmerBaas commented:
Thanks for the feedback. I've changed the referenced version to 5.0.514-6.0.0, due to the new interface member. Also changed the version of the package to 5.0.0. All tests are green.

loekd commented:
I've applied your changes. v5 is now available on NuGet! 
Thanks again.
